egs2/TEMPLATE/asr1/db.sh

```diff
@@ -123,6 +123,7 @@ SPEECHCOMMANDS=downloads
 TOTONAC=downloads
 PRIMEWORDS_CHINESE=downloads
 SEAME=
+SBN=
```

Comment:
I think a more specific name could be better, like BENGALI (if not used already), and point it to downloads, as the other variables in that folder
BENGALI=downloads (this way you'll be able to use it in your data.sh file) (see other comment)